spring-cloud-alibaba-sentinel-zuul/src/main/java/org/springframework/cloud/alibaba/sentinel/zuul/listener/FallBackProviderListener.java

```diff
@@ -0,0 +1,38 @@
+package org.springframework.cloud.alibaba.sentinel.zuul.listener;
+
+import com.alibaba.csp.sentinel.adapter.zuul.fallback.DefaultBlockFallbackProvider;
+import com.alibaba.csp.sentinel.adapter.zuul.fallback.ZuulBlockFallbackManager;
+import com.alibaba.csp.sentinel.adapter.zuul.fallback.ZuulBlockFallbackProvider;
+import org.apache.commons.collections.MapUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.context.event.EventListener;
+
+import java.util.Map;
+
+
+/**
+ * @author tiger
+ */
+public class FallBackProviderListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(FallBackProviderListener.class);
+
+    @EventListener(classes = ApplicationStartedEvent.class)
```

Comment:
Maybe using SmartInitializingSingleton is better, ApplicationStartedEvent is not compatibility in Greenwich.